src/util/errors/http.rs

```diff
@@ -0,0 +1,45 @@
+use std::fmt;
+
+use conduit::Response;
+
+use super::{AppError, Bad, StringError};
+use crate::util::json_response;
+
+#[derive(Debug)]
+pub(super) struct Ok(pub(super) String);
```

Comment:
Why is this `pub(super)`? I assumed the super module can see this anyway, and `pub(super)` is only needed if access from sibling modules is required.

Reply:
Without `pub(super)` I would have to move the `cargo_err`, `server_error`, etc helpers into this file, and reexport them.  I'll probably move everything there eventually, but for now I'd like to keep these helpers and their documentation/comments at the top of `error.rs`.

The parent module cannot see these without the `pub(super)`.  Submodules can see private types/fields from parent modules, but not the other way around.

Reply:
You are right, I got confused there.